ext/ldap/ldap.c

```diff
@@ -2270,7 +2270,7 @@ static void php_ldap_do_modify(INTERNAL_FUNCTION_PARAMETERS, int oper, int ext)
 			ldap_mods[i]->mod_bvalues[1] = NULL;
 		} else {
 			SEPARATE_ARRAY(value);
-			num_values = zend_hash_num_elements(Z_ARRVAL_P(value));
+			int num_values = zend_hash_num_elements(Z_ARRVAL_P(value));
```

Comment:
zend_hash_num_elements returns a uint32_t.

Reply:
Fixed in a follow-up commit